File: repasoHerenciajava/Personal.java
class Personal {
    String name;
    String email;
    String password;
    Integer age;
    public Personal (String name, String email, String password) {
        this.name = name;
        this.email = email;
        this.password = password;
    }
    void printData() {
        System.out.println("My name is " + name);
        System.out.println("and email is " + email);
    }
}

File: repasoHerenciaphp/personal.php
<?php
class Personal {
    public $name;
    public $email;
    public $password;
    public $age;
    function __construct($name, $email, $password) {
        $this->name = $name;
        $this->email = $email;
        $this->password = $password;
    }

    function printData(){
        echo "My name is {$this->name} and my email is {$this->email}\n";
    }
}